flocker/node/functional/test_docker.py

```diff
@@ -367,6 +367,24 @@ def added(_):
         d.addCallback(added)
         return d
 
+    def test_null_environment(self):
+        """
+        A container that does not include any environment variables contains
+        an empty ``environment`` in the return ``Unit``.
+        """
+        image_name = u"clusterhq/buildroot:latest"
```

Comment:
In another branch @exarkun has used an image which is specific to a particular test. The relevant Dockerfile lives besides the test. See the discussion about the cost / benefits of this on https://github.com/ClusterHQ/flocker/pull/1281/files and do similar.